tsconfig.json

```diff
@@ -17,7 +19,6 @@
       "novo-elements": ["dist/novo-elements"],
       "novo-examples": ["dist/novo-examples"],
       "chomsky": ["dist/chomsky"]
-      // comment the above paths and uncomment the following paths to compile with sources
       // "novo-elements": ["projects/novo-elements/src/index"],
       // "novo-examples": ["projects/novo-examples/src/index"],
```

Comment:
was this comment unintentionally removed?

Reply:
This is a comment I added a couple of month ago. I guess I removed it because I thought it would be better to put this kind of information in a README file. But I can add it back. No strong opinion here :)